app/controllers/campaigns_controller.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class CampaignsController < ApplicationController
+  before_action :set_campaign, only: %i[show update]
+
+  # GET /campaigns
+  def index
+    json_response(Campaign.all)
```

Comment:
We should Order by end_date where the one that is ending soonest is the first

Reply:
Added the order, but would we need to filter on `active` also?

Reply:
Added a query parameter for this endpoint